src/attachments/attachments.js

```diff
@@ -0,0 +1,76 @@
+const request = require("request-promise-native");
+
+const randomId = require("../utils/randomId");
+
+const addAttachment = async (
+  apiClient,
+  { taskId, commentText, attachmentUrl, attachmentFileName }
+) => {
+  const comment = {
+    id: randomId(),
+    task_id: taskId,
+    body: commentText,
+  };
+
+  const attachment = {
+    id: randomId(),
+    parent_id: comment.id,
+  };
+
+  const attachmentVersion = {
+    id: randomId(),
+    attachment_id: attachment.id,
+    name: attachmentFileName,
+  };
+
+  try {
+    /* The `request` library is deprecated, however it was the method I was able
+      to use to fetch the file from the URL and send it to the /files endpoint.
+      It may need a refactoring at some point.
+    */
+    const file = await request.get(attachmentUrl);
```

Comment:
I don't get it - wouldn't it be better to only provide a Blob/File API for this? So that this can work both for uploading something we have locally, (and for reuploading something that's already on the internet, as in this example - you just have to fetch it yourself)

Reply:
That'd be better, of course. I felt I've spent too much time in my opinion on this one back then (I've had some problems figuring out the request to make). We can improve it, ofc.